frontend/src/components/Pages/Calendar/RaceSubPages/RaceResults.css

```diff
@@ -0,0 +1,135 @@
+@font-face {
+  font-family: f1Font;
+  src: url('./../../../../assets/fonts/Formula1-Regular.ttf');
+}
+
+.results {
+  font-family: f1Font;
+  color: #ffffff;
+  margin-bottom: 20px;
+}
+
+.results h2 {
+  color: black;
+}
+
+.results table {
+  border-spacing: 0px;
+  padding: 10px;
+  background: #151515;
+  font-size: 14pt;
+  border-radius: 5px;
+}
+
+.results tr:nth-child(odd) {
+  background-color: #151515;
+}
+
+.results tr:nth-child(even) {
+  background-color: #202020;
+}
+
+.results td {
+  vertical-align: middle;
+  margin: 3px 0px;
+}
+
+.results__thead__stops {
+  padding: 0px 15px;
+}
+
+.results__thead__penalties {
+  padding: 0px 15px;
+}
+
+.results__thead__points {
+  padding: 0px 15px;
+}
+
+.results__position {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-color: #ffffff;
+  color: #000000;
```

Comment:
w linijce 13 jest color: black;
Proponuje zawsze używać wartości w Hex'ie

Reply:
Dostosuję się do Twojej uwagi. Uważam jednak, że kiedy da się napisać kolor jest to korzystne, bo nie każdy zna jaki kolor w hex ma jakie oznaczenie. Oczywiście zgadzam się, że w kodzie nie powinny równocześnie występować słowa black i hex mu odpowiadający.